indico/modules/events/abstracts/templates/abstract/judge.html

```diff
@@ -37,44 +37,106 @@
                             'negative': _("No reviewer proposed to accept this abstract"),
                             'mixed': _("Not all reviewers proposed to accept this abstract")} %}
     {% set state = abstract.get_track_reviewing_state(track) %}
-    <span class="i-tag outline {{ mapping_class[state.name] }}" title="{{ mapping_title[state.name] }}">
-        {{ state.title -}}
-    </span>
+    <div class="state-icon {{ mapping_class[state.name] }} flexrow f-a-center f-j-center"
+         title="{{ mapping_title[state.name] }}">
+        <span
+            {%- if state.name in ('positive', 'conflicting') %}
```

Comment:
Why not use another mapping and then just `class="{{ foo.get(state.name, 'icon-circle-small') }}"`?